lib/themes/dosomething/paraneue_dosomething/js/app.js

```diff
@@ -44,26 +44,27 @@ define("app", function (require) {
       Donate.init();
     }
 
-
-    var $mosaicGalleries = $body.find(".gallery.-mosaic");
-
+    var $revealGalleries = $body.find('[data-show-more="true"]');
     /**
      * Show hide large galleries.
-     * Only applies to taxonomy pages for the time being.
      */
-    if ($body.hasClass("page-taxonomy") && $mosaicGalleries.length > 0) {
+    if ($revealGalleries.length > 0) {
       var galleryList = [];
 
-      $mosaicGalleries.each(function (index, gallery) {
+      $revealGalleries.each(function (index, gallery) {
         var $gallery = $(gallery);
         var $tiles = $gallery.find("li");
+        var isMosaicGallery = $gallery.hasClass('-mosaic');
+        var initialItems = isMosaicGallery.hasClass('-mosaic') ? 5 : 6;
+        var stepItems = isMosaicGallery.hasClass('-mosaic') ? 8 : 6;
```

Comment:
Gah! No need to check if isMosaicGallery has a class... that should be a boolean...